bundles/org.openhab.binding.mqtt.generic/src/main/resources/ESH-INF/thing/generic-thing.xml

```diff
@@ -13,6 +13,20 @@
 		<label>Generic MQTT Thing</label>
 		<description>You need a configured Broker first. Dynamically add channels of various types to this Thing. Link
 			different MQTT topics to each channel.</description>
-	</thing-type>
 
+		<config-description>
+			<parameter name="availabilityTopic" type="text">
```

Comment:
What about making this parameters `advanced`? I think most users don't need it and the available option should be kept at a minimum.